### ISSUE TITLE
elasticsearch.keystore Not Created

### ISSUE THREAD

**Issue by KevinCooper** [elastic/stack-docker#91]
After running the setup.yaml, the `elasticsearch.keystore` is not being created.

.env file:
```
PWD=/home/user
ELASTIC_VERSION=6.6.0
TAG=6.6.0
```

output from `sudo docker-compose -f setup.yml up`:
```
setup_1  | Setup completed successfully. To start the stack please run:
setup_1  |       docker-compose up -d
setup_1  |
setup_1  | If you wish to remove the setup containers please run:
setup_1  |      docker-compose -f docker-compose.yml -f docker-compose.setup.yml down --remove-orphans
setup_1  |
setup_1  | You will have to re-start the stack after removing setup containers.
setup_1  |
setup_1  | Your 'elastic' user password is: XYZ
```

output from `sudo docker-compose -f docker-compose-elastic.yml up`:
```
ERROR: for elasticsearch  Cannot create container for service elasticsearch: invalid mount config for type "bind": bind source path does not exist: /home/user/config/elasticsearch/elasticsearch.keystore

ERROR: for elasticsearch  Cannot create container for service elasticsearch: invalid mount config for type "bind": bind source path does not exist: /home/user/config/elasticsearch/elasticsearch.keystore
ERROR: Encountered errors while bringing up the project.
```

**Comment by darnautov** [comment]
I have the same issue, using 7.0.1

**Comment by KevinCooper** [comment]
solved in the issue/PR #89
